cmd/clusterctl/validation/validate_cluster_api_objects_test.go

```diff
@@ -139,7 +139,7 @@ func TestGetClusterObjectWithOneCluster(t *testing.T) {
 	}
 	for _, testcase := range testcases {
 		t.Run(testcase.name, func(t *testing.T) {
-			cluster, err := getClusterObject(c, testcase.clusterName, testcase.namespace)
+			cluster, err := getClusterObject(context.TODO(), c, testcase.clusterName, testcase.namespace)
```

Comment:
We should probably figure out what the right thing to do is when we need a context in tests (TODO at callsite vs TODO-per-test vs somehow bind to the testing.T), but not a blocker for this PR.

Reply:
Sure, we'll try to think about this and raise follow up PR if possible